src/pineko/evolve.py

```diff
@@ -106,6 +106,7 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     """
     # Add a +1 to the orders for the difference in convention between nnpdf and pineappl
     is_fns = int(check.is_fonll_b(tcard["FNS"], pineappl_grid.lumi()))
+    # NB: This would not happen for simFONLL
```

Comment:
since the name `simFONLL` never went outside maybe let's revert to `nFONLL`?

Reply:
[c718a1f](https://github.com/NNPDF/pineko/pull/99/commits/c718a1f4c7b3d352c7ed365c7cac52196b00fe31)